lib/index_group.rb

```diff
@@ -15,6 +15,9 @@ class IndexGroup
     def initialize(base_uri, name, schema, search_config)
       @base_uri = base_uri
       @client = Services::elasticsearch(hosts: base_uri)
+      # Index creation can take longer than other requests, so create a separate
+      # client with a longer timeout
+      @index_creation_client = Services::elasticsearch(hosts: base_uri, timeout: 10)
```

Comment:
Are we happy to do this everywhere, or are we confident enough this is a testing issue (as that is the only place we regularly do multiple slow calls together that we could make the timeout environment specific?